src/routes/+error.svelte

```diff
@@ -1,10 +1,55 @@
 <script lang="ts">
   import { page } from "$app/stores";
+  $: ({ status, error } = $page);
 </script>
 
-<div class="usa-section usa-prose grid-container">
-  {#if $page.error}
-    <h1>{$page.error.title || $page.status}</h1>
-    <p>{$page.error.message}</p>
-  {/if}
-</div>
+<svelte:head>
+  <meta name="robots" content="noindex nofollow" />
+</svelte:head>
+
+<!-- To reduce further error possibilities, we use as much plain HTML here as possible. -->
```

Comment:
There's a couple things I could change here if we don't think my comment above is an issue:
* optimize the error image and generate a blurhash for it
* use `<Button>` components below (using `isLink`).

Reply:
I don't think a 404 happening will be particularly associated with front-end JS errors. We would want to avoid as much as possible in a [`handleError`](https://kit.svelte.dev/docs/hooks#shared-hooks-handleerror) function, but a 404 is an "expected error" so I think it's safe to use our normal components.